### PR TITLE
V2.0 unblockify

### PR DESCRIPTION
**Pull request by jansepar**

### CODE REVIEW

**Comment by noahadams** [comment]
tests!

File: src/unblockify.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    // Grab reference to old insertMobifyScripts method
+    var oldInsert = Capture.prototype.insertMobifyScripts;
+
+    // Override insertMobifyScripts to also move the scripts
+    // to the end of the body
+    Capture.prototype.insertMobifyScripts = function() {
+        oldInsert.call(this);
+
+        var doc = this.capturedDoc;
+        var scripts = scripts || Utils.removeBySelector("script", doc);
```
Comment (noahadams) [inline src/unblockify.js]: This assignment is funky, where would this pre-existing scripts variable be coming from, and aren't we shadowing it in the current function's context anyways?

File: examples/unblockify/index.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            } else {
+                isCompatible = true;
+            }
+        }
+        if (!isCompatible) {
+            return false;
+        }
+        return true;
+    },
+    // path to mobify.js
```
Comment (noahadams) [inline examples/unblockify/index.html:70]: these comments should go before merge

File: src/unblockify.js (hunk 1 above)
Comment (jansepar) [inline src/unblockify.js]: Ah, good catch. In my previous implementation from yesterday I had an optional `scripts` argument. I decided to get rid of that, but left variable hanging around.

**Comment by jansepar** [comment]
@noahadams the eagle has landed! eagle === tests

**Comment by noahadams** [comment]
plz integrate the tests with the runner for CI

**Comment by jansepar** [comment]
@noahadams good call! Added them last night but forgot to commit and push. Done now.

**Comment by noahadams** [comment]
Also, please, get the utils tests passsing with your changes.

**Comment by noahadams** [comment]
All right, this looks like it does what it says. Here you go, a :+1:

**Comment by jansepar** [comment]
Thanks @noahadams!

**Pull request merged**